packages/datagrid/src/textrenderer.ts

```diff
@@ -31,6 +31,7 @@ class TextRenderer extends CellRenderer {
     this.backgroundColor = options.backgroundColor || '';
     this.verticalAlignment = options.verticalAlignment || 'center';
     this.horizontalAlignment = options.horizontalAlignment || 'left';
+    this.overflow = options.overflow || 'ellipsis';
```

Comment:
Should the default be `'clip'`?

Reply:
From an efficiency/cost perspective, it should be clip, but from a default UX experience I feel ellipsis are more user friendly. What do you think?

Reply:
I was going based on what line 335 says.

Reply:
Ah sorry, I was saying that either value as default makes sense, depending
on whether you want to optimize the speed or the UX.